examples/misc_controls_trackball.html

```diff
@@ -149,14 +154,16 @@
 			function animate() {
 
 				requestAnimationFrame( animate );
+				
 				controls.update();
+				
+				stats.update();
```

Comment:
This line was moved so that the FPS stats meter would be correctly updated ever frame, and not just on frames which involve user input (which is required to trigger the `render` function).